ts/packages/node/package.json

```diff
@@ -51,6 +51,7 @@
     "typeorm": "^0.3.16"
   },
   "devDependencies": {
-    "@types/lodash": "^4.14.195"
+    "@types/lodash": "^4.14.195",
+    "keyv": "^4.5.2"
   }
-}
```

Comment:
not sure why is this diff, probably EOL?

Reply:
oh,  probably hahah

Reply:
I'm curious about enforcing a new line at EOF. Which OS are you using? I'm wondering why Prettier isn't catching this, or if there's another tool we might need to incorporate.